apis/database/v1beta1/rdsinstance_types.go

```diff
@@ -430,6 +430,15 @@ type RDSInstanceParameters struct {
 	// +optional
 	MasterUsername *string `json:"masterUsername,omitempty"`
 
+	// MasterPasswordSecretRef references the secret that contains the password used
+	// in the creation of this RDS instance. If a reference is not given, a password
+	// will be auto-generated.
+	//
+	// The key in the secret should be "password"
```

Comment:
Should this be configurable?

Reply:
Yeah, thanks for catching! Changed to `SecretKeySelector`.